### DISCUSSION TITLE
How to sync settings between devices?

### DISCUSSION THREAD

**Discussion by marcinkunert** [Flow-Launcher/Flow.Launcher#1860] [Q&A]
I'm working on 3 devices: PC at the office, PC at home and laptop. What would be the best approach to keep my settings, plugins in sync? I'm pretty new to this, and so far I've discovered portable mode, and moved the directory to Dropbox. Is there a better alternative?

Example scenarios to sync:

- installed new plugin
- changed the shortcut for given functionality
- changed settings for one of the plugins

**Comment by jjw24** [comment]
Hi there, portable mode is the best way to go. We do not provide services to sync data so you will need a third party solution. Dropbox would be a good example.

To keep in sync, you just need to move ur portable to your sync provider. Keep in mind flow preloads settings during startup and when will save settings if you have close the settings window or updated any plugin's settings, so might be good to press F5 in the search window to reload all plugin data before you start using it.

**Comment by jjw24** [comment]
@marcinkunert please let us know how you go as we haven't had many known use cases like this, an area with potential improvements.

**Reply by marcinkunert** [reply]
So far it works mostly fine. There are some issues on autostart though, but the app seems to start anyway, so it doesn't bother me too much. I suspect some of the resources are locked till while the app is running, thus Dropbox is not allowed to replace them. It might be a good idea to keep track about the startup crashes, so I could report them. They might be easy to fix, who knows
